nservicebus/hosting/cloud-services-host/running-without-emulator.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Running locally without Azure Cloud Services emulator
+summary: Debugging endpoints without Azure Cloud Services emulator
+component: CloudServicesHost
+reviewed: 2017-02-21
+---
+
+Debugging endpoints hosted with Cloud Services requires spinning up Azure Cloud Service emulator and loading endpoint in the emulator. It's a heavy operation and if no scale out of roles is required, can be done by loading endpoints using a host process provided by NServiceBus instead.
```

Comment:
isnt "NServiceBus" ambiguous? no host is provided in the NServiceBus nuget. not sure how to re-word though